src/Symfony/Component/Console/Tester/TesterTrait.php

```diff
@@ -162,7 +162,7 @@ private static function createStream(array $inputs)
     {
         $stream = fopen('php://memory', 'r+', false);
 
-        fwrite($stream, implode(PHP_EOL, $inputs));
+        fwrite($stream, implode(PHP_EOL, $inputs).PHP_EOL);
```

Comment:
When an empty is passed, this still sends the PHP_EOL.
Should the code be rewritten as such?
```php
foreach ($inputs as $line) {
    fwrite($stream, $line.PHP_EOL);
}
```